Documentation/op-guide/clustering.md

```diff
@@ -391,7 +391,7 @@ infra2.example.com.  300  IN  A  10.0.1.12
 
 #### Bootstrap the etcd cluster using DNS
 
-etcd cluster members can listen on domain names or IP address, the bootstrap process will resolve DNS A records.
+etcd cluster members can advertise domain names or IP address, the bootstrap process will resolve DNS A records.
```

Comment:
Can we add more details? We can just say something like https://github.com/coreos/etcd/blob/master/Documentation/upgrades/upgrade_3_2.md#change-in---listen-peer-urls-and---listen-client-urls.

Reply:
Yeah makes sense I found that after the fact. Will update to reflect versions.